docs/eventing/broker/kafka-broker/README.md

```diff
@@ -1,8 +1,7 @@
-# Apache Kafka Broker
+# Knative Kafka Broker
 
-The Apache Kafka Broker is a native Broker implementation, that reduces
-network hops, supports any Kafka version, and has a better integration
-with Apache Kafka for the Knative Broker and Trigger model.
+The Knative Kafka Broker is a Apache Kafka native implementation of the Knative Broker API, that reduces
```

Comment:
Building on what @psschwei spotted, I've noticed another tiny thing. Other than that LGTM from me as well.
```suggestion
The Knative Kafka Broker is an Apache Kafka native implementation of the Knative Broker API that reduces
```